### ISSUE TITLE
Continuous effect output options

### ISSUE THREAD

**Issue by GaniatsouElissavet** [brianstock/MixSIAR#334]
Hello Dr. Stock and everyone,

I am working on some isotopic data with MixSIAR and I have a question about the output summary of continuous effects. Sorry if this is posted elsewhere...

My aim is to estimate the proportion of three sources for different individuals using age as a continuous factor. I have 2 tracers (N,C) and I'm running all individuals together as fixed factors. I was able to recreate the plots with the continuous effect for each individual and in the output summary, I can find the proportions of the three sources for each individual. I was wondering if it is possible to extract the proportions for each age (min, median, max ages) of each individual as a txt file. 

Thank you,
Elissavet

**Comment by brianstock** [comment]
Yes you should have access to all the MCMC draws to get the source proportions by age as in the plot. I don't remember the easiest way to do this but try looking at https://github.com/brianstock/MixSIAR/issues/195 and https://github.com/brianstock/MixSIAR/issues/180. There is also some examples of using the MCMC draws directly at http://brianstock.github.io/MixSIAR/articles/modify_output.html.
